src/commands/debug.py

```diff
@@ -59,10 +62,66 @@
 @command("getConfigPlugins")
 @require_channel
 @require_permission(TECHRAT)
-async def cmd_get_plugins(context: Context):
+async def cmd_debug_get_plugins(context: Context):
     """Lists configuration plugins"""
     await context.reply(f"getting plugins...")
 
     plugins = PLUGIN_MANAGER.list_name_plugin()
     names = [plugin[0] for plugin in plugins]
     await context.reply(",".join(names))
+
+
+@command("debug-lastmsg")
+@require_channel
+@require_permission(TECHRAT)
+async def cmd_debug_lastmessage(context: Context):
+    """Get last message sent by user"""
+    specified_user = context.words[1].casefold()
+    await context.reply(f"Last message from {specified_user}:")
+    await context.reply(f"{context.bot.last_user_message[specified_user]!r}")
+
+
+@command("debug-case")
+@require_channel
+@require_permission(TECHRAT)
+async def cmd_create_debug_case(context: Context):
+    debug_rescue = await context.bot.board.create_rescue(client="Shatt", system="HIP 21991",
+                                                         platform=Platforms.PC,
+                                                         active=True,
+                                                         status=Status.OPEN)
+
+    await context.reply(f"Created Debug Case as case #{debug_rescue.board_index}!")
+    await context.reply(f"Client: {debug_rescue.client}    System: {debug_rescue.system}")
```

Comment:
[question] whats up with the excess whitespace in this routine?
if you are trying to space things out please use the string formatting tools. e.g. `{debug_rescue.client: <20}` to left-justify the interpolated string into 20 character width, using spaces as padding.
```py
In [8]: f"{data: <20}"
Out[8]: '21                  '
```